Api/Validators/Restaurants/CreateRestaurantRequestValidator.cs

```diff
@@ -18,6 +18,9 @@ public CreateRestaurantRequestValidator(FileUploadService uploadService, ApiDbCo
             .MaximumLength(50)
             .IsValidName();
 
+        RuleFor(r => r.MaxReservationDuration)
+            .GreaterThanOrEqualTo(30);
```

Comment:
Mi się bardzo nie podoba to że to magiczna liczba, przenieś to proszę do konstanty w klasie Visit, coś w stylu MinReservationDurationMinutes